Add plugin: Quick Tagger

Gorkycreator opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/Gorkycreator/obsidian-quick-tagger

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [x]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
- [x] I have given proper attribution to these other projects in my `README.md`.

joethei commented:
[Copyright (c) 2022 Richard Warner](https://github.com/Gorkycreator/obsidian-quick-tagger/blob/feb497764c5a7a9d341d633c1e830e3273b50aa3/LICENSE#L3)
Update this.

["fundingUrl": "",](https://github.com/Gorkycreator/obsidian-quick-tagger/blob/feb497764c5a7a9d341d633c1e830e3273b50aa3/manifest.json#L9)
Omit this if you don't have such a link.

['Add Tag to Current Note'](https://github.com/Gorkycreator/obsidian-quick-tagger/blob/feb497764c5a7a9d341d633c1e830e3273b50aa3/src/main.ts#L62), ['Remove Tag from Current Note'](https://github.com/Gorkycreator/obsidian-quick-tagger/blob/feb497764c5a7a9d341d633c1e830e3273b50aa3/src/main.ts#L66)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

[s.nameEl.innerHTML = tag.tag_value](https://github.com/Gorkycreator/obsidian-quick-tagger/blob/feb497764c5a7a9d341d633c1e830e3273b50aa3/src/main.ts#L470)
[Avoid using `innerHTML`](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Avoid+%60innerHTML%60%2C+%60outerHTML%60+and+%60insertAdjacentHTML%60)

[console.log(result)](https://github.com/Gorkycreator/obsidian-quick-tagger/blob/feb497764c5a7a9d341d633c1e830e3273b50aa3/src/modal.ts#L94-L96)
Please avoid unnecessary logging. If you need it for debugging purposes, add a check to only log during development.

[let fullId = `obsidian-quick-tagger:${commandId}`](https://github.com/Gorkycreator/obsidian-quick-tagger/blob/feb497764c5a7a9d341d633c1e830e3273b50aa3/src/utilities.ts#L336)
This is outdated.

Gorkycreator commented:
Thanks for your time, I've addressed the issues that were identified and skimmed through for other mistakes. Please let me know if any further changes are needed!
